src/lib/EventEmitter.js

```diff
@@ -0,0 +1,30 @@
+export default class EventEmitter {
+  constructor () {
+    this.subscribers = []
+    return this
```

Comment:
Not sure it is necessary to `return this` in the `constructor()` function of a `class`.

Reply:
Exact :) Thanks